packages/framework/aqueduct/src/data-objects/pureDataObject.ts

```diff
@@ -175,4 +192,42 @@ export abstract class PureDataObject<I extends DataObjectTypes = DataObjectTypes
 	 * Called every time the data store is initialized after create or existing.
 	 */
 	protected async hasInitialized(): Promise<void> {}
+
+	/**
+	 * Called when the host container closes and disposes itself
+	 * @deprecated - 2.0.0-internal.5.2.0 - Dispose does nothing and will be removed in an upcoming release.
+	 */
+	public dispose(): void {
+		super.dispose();
+	}
+
+	/**
+	 * @deprecated - 2.0.0-internal.5.2.0 - PureDataObject does not actually set up to forward events, and will not be an EventForwarder
```

Comment:
```suggestion
	 * @deprecated 2.0.0-internal.5.2.0 - PureDataObject does not actually set up to forward events, and will not be an EventForwarder
```

no dashes after \@deprecated in TSDoc